django-verdant/rca/static/rca/css/streamfield-blocks/showcase.less

```diff
@@ -122,124 +122,125 @@
 
 // SLICK STYLE OVERRIDES
 // Much of this is layout rejigging to account for the large central slide
+.showcase {
```

Comment:
I think we both made the same change here in separate branches, in order to scope the slick styles. The only difference was that I didn't repeat the `.showcase` - yours is perhaps slightly clearer.